test/conformance/api/v1/util.go

```diff
@@ -241,10 +241,12 @@ func validateControlPlane(t pkgTest.T, clients *test.Clients, names test.Resourc
 	t.Log("Checking to ensure Configuration is in desired state.")
 	err = v1test.CheckConfigurationState(clients.ServingClient, names.Config, func(c *v1.Configuration) (bool, error) {
 		if c.Status.LatestCreatedRevisionName != names.Revision {
-			return false, fmt.Errorf("the Configuration %s was not updated indicating that the Revision %s was created: %w", names.Config, names.Revision, err)
+			return false, fmt.Errorf("Configuration(%q).LatestCreatedRevisionName = %q, want %q",
```

Comment:
```suggestion
			return false, fmt.Errorf("Configuration(%s).LatestCreatedRevisionName = %q, want %q",
```
( and ) already create a boundary